clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1496,6 +1496,12 @@ init_thread_in_process(void *drcontext)
             file_type = static_cast<offline_file_type_t>(
                 file_type | OFFLINE_FILE_TYPE_INSTRUCTION_ONLY);
         }
+        file_type = static_cast<offline_file_type_t>(
+            file_type |
+            IF_X86_ELSE(
```

Comment:
This macro has appeared several times now, is it worth sticking into a header?

Reply:
Done.